.changeset/neat-lights-worry.md

```diff
@@ -0,0 +1,22 @@
+---
+"@fluidframework/datastore-definitions": minor
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/test-runtime-utils": minor
+---
+---
+"section": deprecation
+---
+
+The function `process` has been deprecated on the following interface - `IFluidDataStoreChannel`. It has also been deprecated on the following mock test class - `MockFluidDataStoreRuntime`.
+
+A new function `processMessages` has been added in its place which will be called to process multiple messages instead of a single one. This is part of a feature called "Op bunching" where contiguous ops of a given type and to a given data store / DDS are bunched and sent together for processing.
```

Comment:
For the interface case, should we explicitly call out "if you are implementing `IFluidDataStoreChannel` you must now implement `processMessages` too"? The current wording feels like "everything's handled for you" but not sure if that's accurate in the interface case. Not sure if it's actually implemented outside the repo (or supposed to, but it's not `@sealed` so it seems to be allowed).

Reply:
Sure, I will update it to make it explicit.

Reply:
Updated.